homeassistant/components/ecobee/binary_sensor.py

```diff
@@ -1,37 +1,42 @@
 """Support for Ecobee binary sensors."""
-from homeassistant.components import ecobee
-from homeassistant.components.binary_sensor import BinarySensorDevice
+from homeassistant.components.binary_sensor import (
+    BinarySensorDevice,
+    DEVICE_CLASS_OCCUPANCY,
+)
 
-ECOBEE_CONFIG_FILE = "ecobee.conf"
+from .const import DOMAIN
 
 
 def setup_platform(hass, config, add_entities, discovery_info=None):
```

Comment:
Please convert this to the coroutine version `async_setup_platform`. It's a bit faster.